src/posix/sdl/i_input.cpp

```diff
@@ -365,11 +356,33 @@ void MessagePump (const SDL_Event &sev)
 		}
 		break;
 
+	case SDL_MOUSEMOTION:
+	{
+		event.data1 = sev.motion.x;
```

Comment:
With this change events will be sent regardless of `GUICapture`. When it's `true`, they are not sent initially.